app/frontend/src/components/Navigation/ExternalNavButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { Typography, TypographyProps } from "@material-ui/core";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  link: {
+    color: theme.palette.text.secondary,
+    display: "flex",
+    flex: "1",
+    fontSize: "2rem",
+    maxWidth: theme.spacing(21),
+    padding: theme.spacing(1, 1.5),
+    transition: theme.transitions.create(["color", "padding-top"], {
+      duration: theme.transitions.duration.short,
+    }),
+  },
+  label: {
+    alignSelf: "center",
+    marginTop: 0,
+  },
+}));
```

Comment:
I think it's better to pull this out from `NavButton` into its own file and have it called `useLinkStyles` or something similar and import it in both places, considering the styles are literally identical! Otherwise this is error prone for when we need to update styles for links